docs/contribute/redirects.md

```diff
@@ -49,6 +49,19 @@ redirects:
   'testing/redirects/third-page.md':
     anchors:
       'removed-anchor':
+  'testing/redirects/cross-repo-page.md': 'other-repo://reference/section/new-cross-repo-page.md'
+  'testing/redirects/8th-page.md':
+    to: 'other-repo://testing/redirects/5th-page.md'
```

Comment:
With my other comment about removing leniency this should no longer be allowed. We should only link to published remote links.